src/main/java/org/tasks/reservation/Main.java

```diff
@@ -1,24 +1,36 @@
 package org.tasks.reservation;
 
-import java.util.LinkedList;
+import java.io.File;
+import java.io.IOException;
 import java.util.Scanner;
 
 public class Main {
-    public static LinkedList<CoworkingSpace> spaces = new LinkedList<>();
-    private final SpaceManager manager = new SpaceManager();
+    private final static Repository repository = new Repository();
+    private final SpaceManager manager = new SpaceManager(repository);
     private final Scanner scanner = new Scanner(System.in);
-    private final Admin admin = new Admin();
-    private final Customer customer = new Customer();
+    private final Admin admin = new Admin(repository);
+    private final Customer customer = new Customer(repository);
+    private boolean flag = true;
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
         Main main = new Main();
+
+        File file = new File("save.txt");
```

Comment:
use try/catch block (try-with-resources) here

Reply:
@elnurmes, I fixed it in comments.
I replaced class File to FileInputStream, because class File don't implement interface Closable, we can use class FileInputStream for checking of file existence.